index.html

```diff
@@ -2,188 +2,242 @@
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn">See project</button>
+                </div>
+
+                <div class="card bg-2">
+                    <div class="card-content">
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn">See project</button>
+                </div>
+
+                <div class="card bg-3">
+                    <div class="card-content">
+                        <h2>Professional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="btn">See project</button>
                 </div>
-                <button class="btn">See project</button>
             </div>
         </div>
     </section>
 
     <section class="about-section">
         <div class="about-me container">
-            <h2>About Me</h2>
-            <p>
-                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
-                my work and
-                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
-            </p>
-            <button class="btn btn-about">Get my resume</button>
+            <div class="about-me-text">
+                <h2>About Me</h2>
+                <p>
+                    Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t
+                    hesitate to contact me.
+                </p>
+                <button class="btn btn-about">Get my resume</button>
+            </div>
+            <img src="images/illustration-about me@2x.png" alt="" />
         </div>
 
-
+        <hr class="line" />
```

Comment:
- [ ] for this section kindly consider adding the image on the right side as given in the figma design and that will make your project an exact duplicate of the given design check the images below for more clarification 👇 
- your design 👇 
<img width="774" alt="Screen Shot 2023-04-08 at 16 24 26" src="https://user-images.githubusercontent.com/60146030/230726557-d5a1a2d4-3b0c-4770-a1b0-d3ba27ee6e08.png">


- given design 👇 
<img width="670" alt="Screen Shot 2023-04-08 at 16 25 14" src="https://user-images.githubusercontent.com/60146030/230726577-843ec9bb-3783-4aa1-b807-a46659c19c5e.png">

Reply:
I've added the image on the right side.